### PR TITLE
changed email address to web form URL

### PR DESCRIPTION
**Pull request by devbioinfoguy**
for those folks who'd like to leave comments who aren't familiar with Github

### CODE REVIEW

**Comment by ChristinaLK** [comment]
Is this form located in the aci-ref working docs folder?

File: contributing.md [hunk 1]
```diff
@@ -26,5 +26,5 @@ contribution of suggested changes via pull requests. You can do either of the fo
 * [Fork](https://help.github.com/articles/fork-a-repo/) the "[facilitation-best-practices](https://github.com/aci-ref/facilitation_best_practices)" repository to your own GitHub account, make edits, and then submit a "[New Pull Request](https://github.com/aci-ref/facilitation_best_practices/pulls)". The main chapters are in a top-level folder named "activities".
 
 <h3><i>For those less familiar with GitHub</i></h3>
-Please send an email to ACIREF-OUTREACH@lists.clemson.edu to discuss your 
-desired contribution.
+Please fill out our comments & contributions form
+at https://docs.google.com/forms/d/1PbOmG346WowBYh5JQS9POh55VGTbdSXEfaL1kL8Xy8g/viewform.
```
Comment (ChristinaLK) [inline contributing.md]: Can this be made into a link using the `[link name](url)` syntax?

**Comment by devbioinfoguy** [comment]
Sorry about that. URL/Markdown fixed.

**Comment by devbioinfoguy** [comment]
Google sheet and form is in the Best Practices folder (inside ACI-REF working docs)

**Comment by ChristinaLK** [comment]
Perfect.  Thanks @devbioinfoguy!

**Pull request merged**